openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_IncreaseDecrease.java

```diff
@@ -178,17 +178,17 @@ public void execute(StfCoreExtension test, StfSharedClassesExtension sharedClass
 		String[] expectedMessages2 = {"Cache is 100% soft full"};
 		verifyAndPrintCache(sharedClasses, SCSoftmxTestUtil.CACHE_NAME, cacheDir, SCSoftmxTestUtil.CACHE_NAME, 1, expectedMessages2);
 				
+		// Run the third workload (jvm3) in background mode that drives a new workload. 
```

Comment:
This comment is added twice.

Reply:
Thanks for the catch, @pshipton . I've fixed it.